src/vm-vcpu/src/vcpu/mod.rs

```diff
@@ -266,9 +266,12 @@ impl KvmVcpu {
                             {
                                 eprintln!("Failed to write to serial port");
                             }
-                        } else if addr == 0x060 || addr == 0x061 || addr == 0x064 {
-                            // Write at the i8042 port.
-                            // See https://wiki.osdev.org/%228042%22_PS/2_Controller#PS.2F2_Controller_IO_Ports
+                        } else if data.len() == 1 && data[0] == 0xfe && addr == 0x64 {
```

Comment:
I think we should open an issue to not forget about using i8042 from vm-superio once we do a new release there (assuming we are waiting for the metrics to be merged until creating a new release).

Reply:
Do we even need this patch for anything?

Reply:
Sure, I'll add an issue if we merge the PR like this. This commit was needed for the VMM to actually terminate on reboot, or so it seemed when I cherry-picked it.

Reply:
We can update the default kernel command line to use "reboot=t" instead of "reboot=k" until we have a proper keyboard implementation that allows cleaner reboots.

Reply:
Sounds good ... I'll drop this commit then.